discord/guild.py

```diff
@@ -4292,3 +4333,45 @@ async def create_automod_rule(
         )
 
         return AutoModRule(data=data, guild=self, state=self._state)
+
+    @property
+    def invites_paused_until(self) -> Optional[datetime.datetime]:
+        """:class:`datetime.datetime`: When invites get enabled again.
```

Comment:
```suggestion
        """Optional[:class:`datetime.datetime`]: If invites are paused, returns when invites will get enabled in UTC,
        otherwise returns None.
```